File: 17-Problem-RepeatSixteen/17-Problem-RepeatSixteen-TS/17-Problem-RepeatSixteen.js
// Repeat problem 16 to work in the rote memorization of declaring most types, generating random numbers in a range, declaring a functions, using loops, using conditionals, and printing to console.
var nint = Math.floor(Math.random() * (10 - 1) + 1);
var flote = 32.32;
var chr = 'c';
var str = "hello";
var boo = true;
var incrementNint = function (num) {
    if (num > 5) {
        var adder = Math.floor(Math.random() * (10 - 1) + 1);
        var loops = Math.floor(Math.random() * (10 - 1) + 1);
        while (loops > 0) {
            num += adder;
            adder++;
            loops--;
        }
    }
    return num;
};
console.log("nint: ".concat(nint, " flote: ").concat(flote, " chr: ").concat(chr, " str: ").concat(str, " boo: ").concat(boo));
console.log("incrementNint(nint): ".concat(incrementNint(nint)));

File: 17-Problem-RepeatSixteen/17-Problem-RepeatSixteen.swift
// Repeat problem 16 to work in the rote memorization of declaring most types, generating random numbers in a range, declaring a functions, using loops, using conditionals, and printing to console.

let nint: Int = Int.random(in: 1..<10);
let flote: Float = 32.32;
let chr: Character = "c";
let str: String = "hello";
let boo: Bool = true;

func incrementNint(num: Int) -> Int {
    var local_num: Int = num;
    if num > 5 {
        var adder: Int = Int.random(in: 1..<10);
        var loops: Int = Int.random(in: 1..<10);

        while loops > 0 {
            local_num+=adder;
            adder+=1;
            loops-=1;

        }
    }
  return local_num;
}

print("nint: \(nint) flote: \(flote) chr: \(chr) str: \(str) boo: \(boo)");
print("incrementNint(nint): \(incrementNint(num: nint))");
